arviz/plots/backends/bokeh/autocorrplot.py

```diff
@@ -2,17 +2,37 @@
 import bokeh.plotting as bkp
 import numpy as np
 from bokeh.layouts import gridplot
+from bokeh.models import DataRange1d
 from bokeh.models.annotations import Title
 
 from . import backend_kwarg_defaults, backend_show
 from ...plot_utils import _create_axes_grid, make_label
 from ....stats import autocorr
+from ....rcparams import rcParams
 
 
 def plot_autocorr(
-    axes, plotters, max_lag, figsize, rows, cols, line_width, combined, backend_kwargs, show,
+    axes,
+    plotters,
+    max_lag,
+    figsize,
+    rows,
+    cols,
+    line_width,
+    combined,
+    backend_config,
+    backend_kwargs,
+    show,
 ):
     """Bokeh autocorrelation plot."""
+    if backend_config is None:
+        backend_config = {}
+
+    backend_config = {
+        **backend_kwarg_defaults(("bounds_y_range", "plots.bokeh.bounds_y_range"),),
+        **backend_config,
+    }
+
```

Comment:
I think in this case, the priority order should be: user introduced argument (backend_config), autocorr specific default, rcParam value. How does this sound? @ahartikainen @canyon289 

Therefore, in this case I would do:

```
if backend_config is None:
    backend_config = {}

backend_config.setdefault("bounds_range_x", (0, len_y))
```

Then the if is not necessary, now the value in backend_config is the one to be used always.